test/acceptance/accessibility.py

```diff
@@ -243,6 +243,78 @@ def test_multiple_rubrics(self):
         self._check_a11y(self.staff_area_page)
 
 
+class MultipleOpenAssessmentA11yTest(OpenAssessmentA11yTest, FullWorkflowMixin):
+    """
+    Test accessibility when we have a unit containing multiple ORA blocks.
+    """
+
+    def setUp(self):
+        super(MultipleOpenAssessmentA11yTest, self).setUp('multiple_ora', staff=True)
+        self.staff_area_page = StaffAreaPage(self.browser, self.problem_loc)
+
+    # TODO: remove this method override. See TNL-1593
+    def _check_a11y(self, page):
+        page.a11y_audit.config.set_scope(
+            exclude=[
+                ".container-footer",
+                ".nav-skip",
+                "#global-navigation",
+                "#openassessment",
```

Comment:
We don't want to exclude all of these things. Please only exclude duplicate-id, and only for your new test. We want to minimize the number of things being ignored so that new issues are not introduced.